.github/workflows/zxc-execute-otter-tests.yaml

```
@@ -0,0 +1,245 @@
+# SPDX-License-Identifier: Apache-2.0
+name: "ZXC: [Node] Compile Application Code"
```

```suggestion
name: "ZXC: Execute Otter Test"
```

Change title of the workflow?